bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Trajectory/Segment.cpp

```diff
@@ -177,19 +192,21 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Trajectory_Segment(pybind11::module&
 
             )doc"
         )
+
         .def(
-            "compute_delta_v",
-            &Segment::Solution::computeDeltaV,
-            arg("specific_impulse"),
+            "re_compute_states_at",
+            &Segment::Solution::reComputeStatesAt,
```

Comment:
```suggestion
            &Segment::Solution::calculateStatesAt,
```
Nit: I don't think we have to specify the "re" compute here as we are technically doing a new computation at specific timestamps.